src/resources/views/crud/buttons/delete.blade.php

```diff
@@ -5,6 +5,7 @@
 {{-- Button Javascript --}}
 {{-- - used right away in AJAX operations (ex: List) --}}
 {{-- - pushed to the end of the page, after jQuery is loaded, for non-AJAX operations (ex: Show) --}}
+{{-- - cannot use bassetBlock here, because of inline PHP that we WANT rendered (ability to change lang) --}}
```

Comment:
Ok so here's the deal. Here's the problem. This code block... has PHP inside it. And not any kind of PHP... but translations... So YES, we could use `@bassetBlock` here, and that entire script would get internalized... but it would get insternalized with EN translations.

If the developer changes the `app()->getLocale()` to FR, then... at the second call... that file is already internalized... and the messages are in ENGLISH, unfortunately.

Reply:
Later edit. Ok soo... this limitation... the one that I'm talking about... it's not quite a limitation... as much as it is PROOF of how FUCKING AWESOME bassets is. Because instead of saying "_no this can't be done_" what I've done is... created a file for each language 🤯

Because YES, using bassets we can do 
```php
@bassetBlock('backpack/crud/buttons/delete-button-'.app()->getLocale().'.js')
```

Which has downsides:
- this will NOT get internalized using `php artisan basset:internalize`

But it also has upsides:
- we now have that bit of JS separated, for each individual language the admin panel might use:

<img width="770" alt="CleanShot 2023-02-21 at 09 56 35@2x" src="https://user-images.githubusercontent.com/1032474/220282733-7c1c81c8-309f-4ccc-b155-d33738f27a3c.png">

So in the end... I think this is a pretty freakin' good solution. In fact... as I said... I think it's PROOF of how freakin' awesome 🐶 is.

Reply:
So no, this usage does NOT highlight we should keep `@loadOnce` 🎉
But there might be other usages... that do 😔

Reply:
And sure enough, some PRO fields have posed big problems: https://github.com/Laravel-Backpack/PRO/issues/128